src/components/PageResultsNew/BuildMetadata.tsx

```diff
@@ -0,0 +1,181 @@
+/*
+ * This file is part of ciboard
+ *
+ * Copyright (c) 2023 Matěj Grabovský <mgrabovs@redhat.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { useState } from 'react';
+import {
+    Badge,
+    CardBody,
+    DescriptionList,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    Tab,
+    Tabs,
+    TabTitleText,
+} from '@patternfly/react-core';
+import classNames from 'classnames';
+
+import styles from '../../custom.module.css';
+
+import { ExternalLink } from '../ExternalLink';
+
+interface BuildInfoProps {
+    buildId: number;
+    buildTime: string;
+    commit: string;
+    commitTime: string;
+    committerEmail?: string;
+    committerName?: string;
+    owner: string;
+}
+
+function BuildInfo(props: BuildInfoProps) {
```

Comment:
I wonder if this declaration will work as expected, in terms of types.
functional components should have type: `React.FC<BuildInfoProps>`
While this declaration looses correct type.

Reply:
It's not a problem. `React.FC<P>` [is really just a function](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bf33ea3a5a934beb3d4a0e0564e28163c15dcfc9/types/react/index.d.ts#L518-L526) from `P` to a `ReactElement` or `null`. There are some additional properties a functional component might have but they're not required.